src/utils/DateCalculate.js

```diff
@@ -0,0 +1,67 @@
+export function compareDateDifferences(rawDataTime) {
```

Comment:
매개변수 이름은 stringDate 정도로 표현해주면 괜찮지 않을까요?
```suggestion
export function compareDateDifferences(stringDate) {
```